Casks/mx-power-gadget.rb

```diff
@@ -0,0 +1,21 @@
+cask "mx-power-gadget" do
+    version "1.2,20220322"
+    sha256 "702f6ee1e533334bd6ffa42c168b20ad4d39a4c9a197fd0ece7e4b196bfdb929"
```

Comment:
@mavethee Because the `url` is unversioned we cannot provide a shasum.
```suggestion
  sha256 :no_check
```